internal/gw/client.go

```diff
@@ -154,6 +154,11 @@ func (c *client) EnsureUploaded(
 ) error {
 	// TODO: concurrency
 	for _, item := range items {
+		if item.Key == "" && item.LinkTo != "" {
+			log.FromContext(ctx).F("uri", item.SrcPath).Debug("Skipping unfollowed symlink")
+			continue
+		}
+
```

Comment:
Added this too.
I suspected something like this might be necessary but couldn't confirm due to that panic issue.